### ISSUE TITLE
Interface very laggy on Mac OS X iMac retina

### ISSUE THREAD

**Issue by YuriyNasretdinov** [paralect/robomongo#1265]
I found out that on iMac Retina 4K interface is really laggy when I expand window to the full size. When window is relatively small there are not many performance issues. I would say that I get around 10-15 FPS when I open any collection that has some data.

I recorded profiling sample when scrolling the results window to find out where it spends all the time. It looks like text rendering is taking all the time. In the attachment you can see details.
[Sample of Robomongo.txt.zip](https://github.com/paralect/robomongo/files/666423/Sample.of.Robomongo.txt.zip)

**Comment by simsekgokhan** [comment]
Hi @YuriyNasretdinov , thanks a lot for reporting the problem and sharing the profiling results.
We will investigate.

As I see from the file that you are using latest Robomongo version 0.9.0, correct?

**Comment by YuriyNasretdinov** [comment]
Yes, I used the latest available version at the time. I tried older
versions and issue stays the same.
Le jeu. 5 janv. 2017 à 18:21, Gökhan Simsek <notifications@github.com> a
écrit :

**Comment by YuriyNasretdinov** [comment]
I also noticed that Robomongo in 4K window tends to consume a lot of memory when several tabs are open (e.g. 1 Gb). Turning on "low res mode" resolves both performance and memory consumption issues but it looks ugly, of course.
